Makefile

```diff
@@ -65,6 +67,8 @@ install: LDFLAGS += -X "github.com/cockroachdb/cockroach/util.buildTag=$(shell g
 install: LDFLAGS += -X "github.com/cockroachdb/cockroach/util.buildTime=$(shell date -u '+%Y/%m/%d %H:%M:%S')"
 install: LDFLAGS += -X "github.com/cockroachdb/cockroach/util.buildDeps=$(shell GOPATH=${GOPATH} build/depvers.sh)"
 install:
+	@echo "GOPATH set to $$GOPATH"
+	@echo "PATH set to $$PATH"
```

Comment:
Dumping the entire PATH is pretty noisy, and doesn't make it clear what's going on (the user will see a bunch of stuff that is perfectly normal and doesn't have anything to do with us). Instead of doing that you could say "$$GOPATH/bin added to PATH"